src/d3-cloud.directive.js

```diff
@@ -37,173 +37,191 @@
       scope: {
         events: '=?',
         font: '@',
-        ignoreList: '=?',
+        filterFunction: '=?',
```

Comment:
Please preserve ignoreList for backwards compatibility. And I like short attribute names, so how about just `filter`? Could you also use the `&?` binding for the filter function?